csrankings-p.csv

```diff
@@ -665,6 +665,7 @@ Peter McBurney,King's College London,https://www.kcl.ac.uk/people/peter-mcburney
 Peter Michael Fischer,University of Augsburg,https://www.uni-augsburg.de/de/fakultaet/fai/informatik/prof/dbis/team/fischer,n2w2PW8AAAAJ
 Peter Michael Garraghan,Lancaster University,https://www.lancaster.ac.uk/scc/about-us/people/peter-garraghan,lkP_Zd8AAAAJ
 Peter Milder,Stony Brook University,https://www.stonybrook.edu/commcms/electrical/people/-core_faculty/milder_peter,tZ_do3gAAAAJ
+Peter Minary,University of Oxford,https://www.cs.ox.ac.uk/people/peter.minary/,ruz8HrEAAAAJ

```

Comment:
Title is "Research Lecturer"; not clear if this meets inclusion criteria.

Reply:
I think he meets the criteria. We don't have "tenure track" in the UK; but AFAIK he has a permanent contract. He supervises doctoral students, both jointly and solely; for example, these four are sole supervisees:

http://www.cs.ox.ac.uk/people/lyle.hopkins/
http://www.cs.ox.ac.uk/people/jeffrey.mak/
http://www.cs.ox.ac.uk/people/furkan.ozden/
http://www.cs.ox.ac.uk/people/florian.stoertz/